### DISCUSSION TITLE
Saleor integration with Django

### DISCUSSION THREAD

**Discussion by Yaso2Go** [saleor/saleor#16615] [Q&A]
I have an existing Django project that I've been working on, and I'm interested in integrating the Saleor dashboard as an app within this project. Ideally, I would like to access the Saleor dashboard through a route like backend/.

Additionally, I want to connect the Saleor dashboard with my existing product database. I already have a fully responsive frontend built with HTML and CSS that displays products and other related information.

My goal is to integrate all the features of Saleor, such as checkout, payment processing, and other e-commerce functionalities, into my current project without needing to start a new project. How can I achieve this integration while maintaining my existing frontend and backend structure?

Any guidance or suggestions on how to approach this would be greatly appreciated!

**Comment by maarcingebala** [comment]
Hey, I don't think there is an easy solution to do that, as Saleor is a standalone project, not designed to be extended this way. Also, the backend is hosted separately from the front end (the dashboard is a separate front-end app). 

The dashboard fetches all the data via API. If you want to include data from another Django project, you could expose an API in your project and fetch data from multiple APIs in the dashboard (after customizations). Another option to explore could be to create a [GraphQL federation](https://www.apollographql.com/docs/federation/), that merges two GraphQL APIs - Saleor and your data (you would have to add GraphQL API to your service), and then customize the dashboard to consume data from this federation.
